packages/url_launcher/ios/Classes/UrlLauncherPlugin.m

```diff
@@ -165,4 +151,23 @@ - (void)closeWebViewWithResult:(FlutterResult)result API_AVAILABLE(ios(9.0)) {
   result(nil);
 }
 
+- (UIViewController *)topViewController {
+  return [self topViewController:[UIApplication sharedApplication].keyWindow.rootViewController];
+}
+
+- (UIViewController *)topViewController:(UIViewController *)rootViewController
```

Comment:
Add comments on what this method trying to do.
And maybe rename it as `topViewControllerFromViewController:`?